src/TRestRawSignal.cxx

```diff
@@ -631,21 +631,66 @@ void TRestRawSignal::GetSignalSmoothed(TRestRawSignal* smthSignal, Int_t averagi
 /// \param averagingPoints It defines the number of neightbour consecutive
 /// points used to average the signal
 ///
-std::vector<Float_t> TRestRawSignal::GetSignalSmoothed(Int_t averagingPoints) {
+/// \param option If the option is set to "EXCLUDE OUTLIERS", points that are too far away from the median baseline will be ignored to improve the smoothing result
+///
+std::vector<Float_t> TRestRawSignal::GetSignalSmoothed(Int_t averagingPoints, std::string option) {
+    
```

Comment:
Don't quite understand what is this function doing, for me looks different than standard smoothing

Reply:
You mean with the exclude outliers option? Previously pulses were affecting the smoothing, now not anymore

> Old `TRestRawSignal::GetBaseLineSmoothed`: ![GetBaselineSmoothed_old](https://user-images.githubusercontent.com/77107394/168097813-78f35a47-4f88-4d1f-9d2b-56ec5e45259d.png)
> 
> New `TRestRawSignal::GetBaseLineSmoothed` with "EXCLUDE OUTLIERS" option: ![GetBaselineSmoothed_improved](https://user-images.githubusercontent.com/77107394/168097981-f65566bc-7953-4f6f-aad3-7b3bd0916cae.png)

Reply:
I meant the default option

Reply:
Yes, there is the default `GetBaseLineSmoothed`, which was already before overloaded, so you get either a `TRestRawSignal` or a vector returned. I added the new option only to the vector version, since it is the one I use in `GetBaseLineCorrected`